trustyai_tests/tests/explainers/kserve/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+@pytest.fixture(scope="class")
+
+def bank_churn_model(client, model_namespace):
```

Comment:
Have you tried to deploy this model similar to how we deploy other models in drift and fairness testing? i.e.
```
from ocp_resources.inference_service import InferenceService

def bank_churn_model(...):
    with InferenceService(...) as inference_service:
        yield inference_service
```